ci/conda_env_gandiva.txt

```diff
@@ -15,5 +15,5 @@
 # specific language governing permissions and limitations
 # under the License.
 
-clang>=11
-llvmdev>=11
+clang>=11,<19
+llvmdev>=11,<19
```

Comment:
FYI: We can remove this by merging GH-44233.

Reply:
@kou Which one would you rather merge first?

Reply:
I've merged  GH-44233.
I'll update this branch.